packages/elements/src/collapse/index.ts

```diff
@@ -141,7 +148,7 @@ export class Collapse extends BasicElement {
    * @returns {boolean} true if target is ef-header
    */
   private static isHeader (element: HTMLElement): boolean {
```

Comment:
I guess this function is here to prevent hiding when the click happens on the slotted element. If so, why not to create a local reference to the `ef-header` itself?

Reply:
As well, `handleTap` should not be an arrow function.

Reply:
Yes, you understand `isHeader` correctly but I am not getting about `create a local reference` Can you provide more detail?

Reply:
```
html`<ef-header ${ref(this.headerRef)}`

if (target === this.headerRef.value! || target === other-clickable-targets) { ... }

```

Reply:
Thanks

Reply:
We do not want this function anymore, we let developers handle events from content in slots by themselves.